### ISSUE TITLE
FNX-5417 ⁃ Latest string additions need update for exposing to localizers

### ISSUE THREAD

**Issue by Delphine** [mozilla-mobile/fenix#2042]
I've reviewed most recent string additions/changes from [Quarantine PR #22](https://github.com/mozilla-l10n/android-l10n/pull/22/) and have found a few things that need to be fixed in order for us to expose this on Pontoon.

[ ] "browser_menu_library": This should be removed entirely from the file so that it doesn't appear in Pontoon
[ ] "create_collection_save_to_collection_empty": Nit - "stepof"
[ ] "create_collection_save_to_collection_full": "%d" needs to be explicitly called out and explained in comments
also
[ ] "create_collection_save_to_collection_full": Nit - "stepof"

What's really blocking us here aren't the nits, rather the two other issues.
(CC @boek so he's aware)

**Comment by Delphine** [comment]
I was told that these strings weren't final because Collections is still a WIP.
We agreed that strings landing on master were as close to final as possible, because we expose them to localization once they land there.
Please use a separate branch if strings are still a WIP! thanks :)

**Comment by Delphine** [comment]
Betty mentioned these strings were now final, so we can just get the fixes in and expose to l10n

**Comment by Delphine** [comment]
Filed a follow-up issue (#2334) for one remaining thing, that did not block us anymore from exposing on Pontoon.
Thanks so much for your help on this!
